examples.md

```diff
@@ -0,0 +1,105 @@
+# Mano Simulator Examples
+Here are some simple examples you can do with Mano simulator. You can learn how to write a mano program by taking examples from these codes.
+
+### Change RESULT variable to !X
+
+```Assembly
+ORG 0
+LDA X
+CMA
+STA RESULT
+HLT
+X, HEX F0F0
+RESULT, HEX 0000
+```
```

Comment:
Add what the output/result is of each program after the code block. I see you already added the result for some of the examples so just move those below the code.